xbmc/addons/PVRClient.cpp

```diff
@@ -2071,3 +2093,42 @@ void CPVRClient::OnPowerSavingDeactivated(void)
   }
   catch (std::exception &e) { LogException(e, __FUNCTION__); }
 }
+
+PVRError CPVRClient::TranslateError(PVR_ERROR error)
```

Comment:
Was only added to see a way and to ask about. Not wanted to do all changes and then to bad for you :)